doc/gov/chapters/chapter01.md

```diff
@@ -43,11 +43,11 @@ Covers the processes for nominating and selecting candidates to be leads and co-
 
 * Consensus-Based Decision Making (Ch. 5)
 
-The approaches and processes by which CNTT contributors shall use to ensure fair representation for key decisions made, approve releases; including how to manage dissention. 
+The approaches and processes by which CNTT contributors shall use to ensure fair representation for any key decisions and approve release approvals. This includes mechanisms on how to manage dissention. 
```

Comment:
Can we simplify: "The approaches and processes by CNTT contributors shall use ..." 

nit pick: "dissention" is a rarely used spelling for "dissension"